### PR TITLE
Ignore string prefixes like "b" or "u"

### PR DESCRIPTION
**Pull request by xZise**
When the string uses prefixes, the checker wasn't checking those strings, because it only handles strings that start with a bad quote. To handle this, it gets the last character of the string (which is the quote character used) and removes all characters in front of the first quote character.

It also adds a few tests using one or two prefixes.

This is an implementation for #28.

### CODE REVIEW

File: flake8_quotes/__init__.py [hunk 1]
```diff
@@ -77,15 +77,19 @@ def get_quotes_errors(self, file_contents):
                 # ignore non strings
                 continue
 
-            if not token.string.startswith(self.inline_quotes['bad_single']):
+            # Remove any prefixes in strings like u"foobar". Use the quote
+            # character at the end of the string to know where the prefixes end
+            text = token.string[token.string.index(token.string[-1]):]
```
Comment (twolfson) [inline flake8_quotes/__init__.py]: There is a lot going on in this line. Can we break it down into other variables to make it easier to follow?

Comment (twolfson) [inline flake8_quotes/__init__.py]: Also, comment examples would be great. For example:

```python
# character at the end of the string to know where the prefixes end
#   `"a"` -> `a`
#   `u"a"` -> `a`
#   `ur"a"` -> `a`
```

**Comment by twolfson** [comment]
At a high level, this looks good. Left some notes on how to improve the PR for future maintenance

File: flake8_quotes/__init__.py (hunk 1 above)
Comment (xZise) [inline flake8_quotes/__init__.py]: Is possible, do you want it in a separate commit or should I squash it into this one? Also a small note: Obviously this doesn't obliterate the quotes, only the prefix (e.g. the result would be always `"a"`.

Comment (twolfson) [inline flake8_quotes/__init__.py]: That's why I wanted the comment ;) Please add/squash it to the current commit.

**Comment by xZise** [comment]
Okay squashed it into one commit and this should address your comments you had.

File: flake8_quotes/__init__.py [hunk 2]
```diff
@@ -77,15 +77,22 @@ def get_quotes_errors(self, file_contents):
                 # ignore non strings
                 continue
 
-            if not token.string.startswith(self.inline_quotes['bad_single']):
+            # Remove any prefixes in strings like `u"foo"`. Use the quote
+            # character at the end of the string to know where the prefixes end
+            quote_char = token.string[-1]  # Does only contain one character!
+            quote_start = token.string.index(quote_char)
+            text = token.string[quote_start:]
+            # `"foo"`, `b"foo"`, `br"foo"` all result in `"foo"`
```
Comment (twolfson) [inline flake8_quotes/__init__.py]: This should likely go at the head with the rest of the comments and we should mention that it's `text's` value but to avoid wasting even more time I will land this + make the minor edits.

**Comment by twolfson** [comment]
This has been merged/released in `0.4.0`. I went a little trigger happy with shuffling comments/renaming variables to be more semantic.
